.gitmodules

```diff
@@ -28,6 +25,9 @@
 [submodule "lib/libjpeg/libjpeg-turbo"]
 	path = lib/libjpeg/libjpeg-turbo
 	url = https://github.com/libjpeg-turbo/libjpeg-turbo.git
-[submodule "lib/sdl/SDL_image"]
+[submodule "lib/sdl/SDL_ttf"]
+	path = lib/sdl/SDL_ttf
+	url = https://github.com/libsdl-org/SDL_ttf.git
```

Comment:
For easier review:

- Old: https://github.com/SDL-mirror/SDL_ttf/tree/8f8753c4288948d1fd483d7de23e36c7b30a7807
- New: https://github.com/libsdl-org/SDL_ttf/tree/200901a8f55bcf7aa031157f53a575c942d2b7aa

Looks good.

Worth noting: this is between version 2.0.14 and 2.0.15
We should consider updating to 2.0.15 in a separate step later.